### PR TITLE
Adds Release Notes 4.8.20

### PR DESCRIPTION
**Pull request by darshan-nagaraj**
No BZ.

Preview link: https://deploy-preview-38782--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-20

To be merged only after the erratas are shipped live.

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2865,3 +2865,25 @@ link:https://access.redhat.com/solutions/6487101[{product-title} 4.8.19 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-8-20"]
+=== RHBA-2021:4574 - {product-title} 4.8.20 bug fix update
+
+Issued: 2021-10-15
```
Comment (kalexand-rh) [inline release_notes/ocp-4-8-release-notes.adoc:2872]: Is this the correct date?

**Review by kalexand-rh — COMMENTED** [review]
I have one question, but this is looking good!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (darshan-nagaraj) [reply release_notes/ocp-4-8-release-notes.adoc:2872]: Nope. It should be 2021-10-16. I will change it. Thanks for catching it.

Comment (darshan-nagaraj) [reply release_notes/ocp-4-8-release-notes.adoc]: Done! It's updated now.

Comment (kalexand-rh) [reply release_notes/ocp-4-8-release-notes.adoc]: This looks good! Let me know when you're ready for it to be merged.

**Pull request merged**